### DISCUSSION TITLE
Increment reply details

### DISCUSSION THREAD

**Discussion by smtdev** [mayooear/ai-pdf-chatbot-langchain#49] [Q&A]
Is there any way (maybe a parameter or so) to get a more detailed reply when chatting? I don't get it even if I ask for it in my question.

Thank you for this amazing work!

**Comment by mayooear** [comment]
You can specify your requirements in the prompt in the `QA_PROMPT` inside the `makechain.ts` file. You can emphasise to the LLM to "respond in as much detail as possible" or whatever you require.
